docs/types/mutations.rst

```diff
@@ -10,14 +10,15 @@ This example defines a Mutation:
 
 .. code:: python
 
-    import graphene
+    from graphene import Mutation, String, Boolean, Field
 
-    class CreatePerson(graphene.Mutation):
+
+    class CreatePerson(Mutation):
         class Arguments:
-            name = graphene.String()
+            name = String()
 
-        ok = graphene.Boolean()
-        person = graphene.Field(lambda: Person)
+        ok = Boolean()
+        person = Field(lambda: Person)
 
         def mutate(self, info, name):
```

Comment:
```suggestion
        def mutate(root, info, name):
```

Reply:
This is suggesting the `root` change? Good catch, will fix on second pass.

Reply:
I made a bunch of changes related to `root` argument in this PR as well: https://github.com/graphql-python/graphene/pull/969

Please nitpick, comment, whatever all you want. That PR and this one are going to conflict in ways that will be annoying to resolve, I'm sure.

Reply:
Just a thought, but perhaps we ahould look at the two, and consider which will make the most likely to conflict changes and stage them in order to minimize the conflicts? That is, if one makes minor syntactical changes and the other makes bigger changes to actual layout and wording...?

Reply:
I think we we can resolve the conflicts easily enough, but maybe if we can groom some stories around documentation there'll be less overlap.

Reply:
I think that every method where `root` or `parent` is used as the first argument it should be decorated with `@staticmethod`. Especially if that argument is not an instance of the class. Current approach is not too pythonic.